### ISSUE TITLE
Keyboard shortcuts are not working in Forefox Developer Edition

### ISSUE THREAD

**Issue by hfzkazinazmul** [itsecurityco/to-google-translate#113]
I'm using win 10. In Firefox, all shortcuts are going well. But in Firefox Developer edition, none of them are working.
Shortcuts list that are not working:
1. ctrl+shift+1
2. ctrl+shift+2
3. ctrl+shift+3

**Comment by KrzysztofGA** [comment]
Does your windows have more than one language installed? I have several languages installed and I noticed that **Ctrl + Shift** shortcut is responsible for **"Switch keyboard layout"** and here is the cause of the error. I have the same problem. Is it possible to change keyboard shortcuts?

**Comment by KrzysztofGA** [comment]
> I'm using win 10. In Firefox, all shortcuts are going well. But in Firefox Developer edition, none of them are working.
> Shortcuts list that are not working:
> 
>     1. ctrl+shift+1
>     2. ctrl+shift+2
>     3. ctrl+shift+3

@hfzkazinazmul Does your windows have more than one language installed? I have several languages installed and I noticed that **Ctrl + Shift** shortcut is responsible for **"Switch keyboard layout"** and here is the cause of the error. I have the same problem. Is it possible to change keyboard shortcuts?

**Comment by hfzkazinazmul** [comment]
Yes. there are three installed language in my windows PC. The mentioned shortcuts are properly responsible in Firefox while not working in Firefox developer edition.
And you may find the answer of your last question here:
https://www.isumsoft.com/windows-10/switch-between-inpute-language-on-windows-10.html
https://support.microsoft.com/en-us/help/12445/windows-keyboard-shortcuts

**Comment by KrzysztofGA** [comment]
@hfzkazinazmul You are great :) Thank you
